### PR TITLE
GitHub actions widget

### LINKED ISSUE #1523: github-actions: build at-a-glance widget
Create a card component, for inclusion into for example an entity overview page, showing the build status at a glance for the project it belongs to. The card could for example show the status of the latest master build, in a small amount of space.

### PR DESCRIPTION
**Pull request by shmidt-i**
Fixes #1523 .
Built on top of #1696 .
Still some pieces of the puzzle are missing, e.g. how to inject this widget into the entity view with the arguments needed. 
#1536

### CODE REVIEW

**Comment by stefanalund** [comment]
No screenshots? 😭😬

**Comment by shmidt-i** [comment]
This is just a component that accepts owner and repo and renders github actions table inside, it cannot be consumed atm in the entity view due to some unsolved questions, more like preparations for the future ;)

File: app-config.yaml [hunk 1]
```diff
@@ -20,5 +20,9 @@ proxy:
 organization:
   name: Spotify
 
+github-actions:
+  owner: CircleCITest3
+  repo: try-ssr
```
Comment (Rugvip) [inline app-config.yaml:25]: Don't see the point of having this here tbh? 😅 , might as well just hardcode it in the plugin as it's gonna have to vary for every entity anyway.

File: plugins/catalog/src/routes.ts [hunk 2]
```diff
@@ -25,6 +25,6 @@ export const rootRoute = createRouteRef({
 });
 export const entityRoute = createRouteRef({
   icon: NoIcon,
-  path: '/catalog/:kind/:optionalNamespaceAndName/',
+  path: '/catalog/:kind/:optionalNamespaceAndName/*',
```
Comment (Rugvip) [inline plugins/catalog/src/routes.ts:28]: Is this really needed? O_o, didn't think you needed exact matches in react-router v6 anymore

File: plugins/github-actions/src/components/Widget/Widget.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ * limitations under the License.
+ */
+import React from 'react';
+import { MemoryRouter, Routes, Route } from 'react-router-dom';
+import { buildRouteRef, rootRouteRef } from '../../routes';
+import { WorkflowRunDetailsPage } from '../WorkflowRunDetailsPage';
+import { WorkflowRunsTable } from '../WorkflowRunsTable';
+
+export const GithubActionsWidget = () => (
+  <MemoryRouter initialEntries={[rootRouteRef.path]}>
```
Comment (Rugvip) [inline plugins/github-actions/src/components/Widget/Widget.tsx:23]: Sup with the router here? O_o

File: app-config.yaml (hunk 1 above)
Comment (shmidt-i) [reply app-config.yaml:25]: Going to just recreate this branch from scratch :) Too much leftovers from underlying branch

**Pull request closed without merging**